enlighten/Multispec.py

```diff
@@ -89,6 +87,7 @@ def __init__(self,
         self.in_process = {} # a dict of device_id to WasatchDeviceWrapper (may be None if "gave up")
         self.disconnecting = {}
         self.ignore = {}
+        self.seen_serial_numbers = {} # dict of serial_number to...not sure yet.  Should include initial assigned color
```

Comment:
Might use this later to persist the first color code assigned to each serial number across a session